src/bp-core/admin/bp-core-admin-settings.php

```diff
@@ -836,6 +835,8 @@ function bp_feed_settings_callback_post_type( $args ) {
 	}
 	?>
 	<input
+		class="bp-feed-post-type-checkbox <?php echo 'bp-feed-post-type-' . $post_type; ?>"
```

Comment:
Use the escaping functions in attribute.

Reply:
Done escaping. https://github.com/buddyboss/buddyboss-platform/pull/2290/commits/c7f977ae62f4a3a11cbc6bfed6f72c882fb8b866